cmd/juju/status/status.go

```diff
@@ -403,29 +402,41 @@ func (c *statusCommand) runStatus(ctx *cmd.Context) error {
 	return nil
 }
 
-// clearScreen removes any character from the terminal
-// using ANSI scape characters.
-func clearScreen() {
-	fmt.Printf("\u001Bc")
+// statusArgsWithoutWatchFlag returns all args cut off '--watch' flag of status commands
+// and the value of '--watch' flag
+func (c *statusCommand) statusArgsWithoutWatchFlag(args []string) ([]string, string) {
```

Comment:
You already have the watch value in `c.watch` no need to return it. You only need the stripped args.